sites/partners/src/pages/listings/[id]/lottery.tsx

```diff
@@ -198,6 +198,10 @@ const Lottery = (props: { listing: Listing | undefined }) => {
         (event) => event.type === ListingEventsTypeEnum.publicLottery
       )
       if (listing.lotteryStatus === LotteryStatusEnum.releasedToPartners) {
+        // reverse array to find most recent release date
+        const lotteryReleaseDate = lotteryActivityLogData
+          ?.reverse()
+          ?.find((logItem) => logItem.status === LotteryStatusEnum.releasedToPartners)?.logDate
```

Comment:
Alternative to using .findLast since it was causing build errors. Seems like it was only added recently so our versions may not be quite updated for it https://github.com/microsoft/TypeScript/issues/48829

Reply:
Ope we need typescript v 5 😮

Reply:
This is a great solution. `findLast` is definitely the best option, an alternative could be to filter and grab the last one. I don't know which is more performant, but either way it would be a minuscule difference
```
      const filtered = lotteryActivityLogData?.filter(
        (logItem) => logItem.status === LotteryStatusEnum.releasedToPartners
      )
      const lotteryReleaseDate = filtered[filtered.length - 1]
 ```